java/statistical-query-group-sort/src/main/res/values/strings.xml

```diff
@@ -0,0 +1,15 @@
+<resources>
+    <string name="app_name">Statistical Query (Group and Sort)</string>
+    <string name="add">Add</string>
+    <string name="type">Type:</string>
+    <string name="field">Field:</string>
+    <string name="remove_statistic">Remove Statistic</string>
+    <string name="left"><![CDATA[<<]]></string>
+    <string name="right"><![CDATA[>>]]></string>
+    <string name="group_fields">Group Field(s):</string>
+    <string name="order_by_field">Order by Field</string>
+    <string name="get_statistics">Get Statistics</string>
+    <string name="change_sort_order">Change sort order</string>
```

Comment:
Maybe "Reverse sort order"? I think the button should also be disabled if nothing is selected. Also, when I click it, it looks like the ascending/descending is flipping back and forth a couple times before being set.